0008-quilt-ethos.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+
+Quilt has a set of Community Guidelines and Governance RFCs. These provide a set of rules, expectations and guidance on what and how certain aspects of the project should be managed. After some reflection, it seems that a set of "Values" had not been define; the aim of this document is to set these values.
+
+# Motivation
+
+This can be seen as a derivate or inspiration from the Python Projects [PEP20](https://www.python.org/dev/peps/pep-0020/). I (ToffeeMax) have found this document useful when trying to both write better code using Python and as a general set of development principles. PEP20 could partially be seen as non-serious, however it is referenced within Python frequently. 
```

Comment:
This document feels like it's having a bit of an identity crisis. PEP 20 is about project (and by extension Python programmers) goals, and this document kinda tries to roll project and community principles into one.

Reply:
I for sure agree with you here - I have pushed some changes and removed the "community principles" (mostly)